File: javascript/divisao.js
const n = parseInt(prompt("Quantos casos voce vai digitar?"));

let numerador, denominador, divisao;

for (i = 0; i < n; i++) {
    numerador = parseFloat(prompt("Entre com o numerador:"));
    denominador = parseFloat(prompt("Entre com o denominador:"));

    if (denominador == 0) {
        console.log("DIVISAO IMPOSSIVEL");
    } else {
        divisao = numerador / denominador;
        console.log("DIVISAO = %.2f", divisao);
    }
}

File: python/divisao.py
n = int(input("Quantos casos voce vai digitar? "))

for i in range(0, n):
    numerador = float(input("Entre com o numerador: "))
    denominador = float(input("Entre com o denominador: "))

    if denominador == 0:
        print("DIVISAO IMPOSSIVEL")
    else:
        divisao = numerador / denominador
        print("DIVISAO =", divisao)
